src/swift/Data.swift

```diff
@@ -10,43 +10,55 @@
 //
 //===----------------------------------------------------------------------===//
 
-public struct DispatchData : RandomAccessCollection, _ObjectiveCBridgeable {
+import CDispatch
+
+public struct DispatchData : RandomAccessCollection {
 	public typealias Iterator = DispatchDataIterator
 	public typealias Index = Int
 	public typealias Indices = DefaultRandomAccessIndices<DispatchData>
 
 	public static let empty: DispatchData = DispatchData(data: _swift_dispatch_data_empty())
 
+#if false /* FIXME: dragging in _TMBO (Objective-C) */
 	public enum Deallocator {
 		/// Use `free`
 		case free
 
+#if HAVE_MACH
```

Comment:
You're probably right.  I was mentally mapping this deallocator to DISPATCH_DATA_DESTRUCTOR_VM_DEALLOCATE which is guarded by HAVE_MACH in data.c.  It probably makes sense to implement similar functionality via munmap on Linux.